### ISSUE TITLE
[Accessibility] Voiceover is reading entire file once it reaches the end on macOS

### ISSUE THREAD

**Issue by adincebic** [microsoft/vscode#104645]
Issue Type: <b>Bug</b>

## Steps to reproduce
1. Create new file (e.g. test.js)
2. Use command+shift+E to switch over to the source editor
3. Press ctrl+option+shift+down arrow to interact with the text
4. Write some code
    let str = "Hello";
    Console.log(str);
5. Use down arrow to navigate the file content
6. Press down arrow repeatedly to reach the end of file

## Expected result
Voiceover should have read "new line" and produce the usual sound when it reaches the end. Further more, it should not have start reading entire file (top to bottom).

## Actual result
Voiceover starts reading whole file (top to bottom.)

VS Code version: Code 1.48.0 (db40434f562994116e5b21c24015a2e40b2504e6, 2020-08-13T08:03:27.059Z)
OS version: Darwin x64 19.6.0

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i5-7360U CPU @ 2.30GHz (4 x 2300)|
|GPU Status|2d_canvas: enabled<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>metal: disabled_off<br>multiple_raster_threads: enabled_on<br>oop_rasterization: disabled_off<br>protected_video_decode: unavailable_off<br>rasterization: enabled<br>skia_renderer: disabled_off_ok<br>video_decode: enabled<br>viz_display_compositor: enabled_on<br>viz_hit_test_surface_layer: disabled_off_ok<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|1, 3, 6|
|Memory (System)|8.00GB (0.07GB free)|
|Process Argv|-psn_0_15994688|
|Screen Reader|yes|
|VM|0%|
</details><details><summary>Extensions (1)</summary>

Extension|Author (truncated)|Version
---|---|---
csharp|ms-|1.22.1

</details>

**Comment by isidorn** [comment]
@adincebic thanks for filling this. Sounds somewhat similar to https://github.com/microsoft/vscode/issues/89231
And I believe it is a chrome / voice over issue.
So first of all we should figure out if this can be reproduced just with Chrome. So do you encounter this behavior when just using Chrome, for example when navigating this GitHub input box?

**Comment by jhomme** [comment]
Hi. I was going to file a different issue, but I want to comment here and take a stab at what I think is happening. I do not think VoiceOver realizes that the edit window is an edit window. I wonder if explicitly  setting the edit window to a multi-line edit field will cure this issue and the one I am having. I've seen this kind of behavior on PC when JAWS and NVDA do not realize they are dealing with an edit field. The article at the below link talks about this.

It's preferable to use a <textaria> tag, but I don't know how the VS-code stuff works, so I'll quit blabbing and paste the link.

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/textbox_role

**Comment by isidorn** [comment]
@jhomme thanks for the suggestion but what we currently use is a `textarea`. So it seems to cover that.

**Comment by jhomme** [comment]
Hi,
What attributes?

On Aug 28, 2020, at 2:21 PM, Isidor Nikolic <notifications@github.com> wrote:


@jhomme <https://github.com/jhomme> thanks for the suggestion but what we currently use is a textarea. So it seems to cover that.

**Comment by isidorn** [comment]
@jhomme no attributes, a [`textarea`](https://www.w3schools.com/tags/tag_textarea.asp) is a native html element.

**Comment by isidorn** [comment]
Closing as dup of  #89231
